Detectors/EMCAL/workflow/include/EMCALWorkflow/CalibLoader.h

```diff
@@ -146,55 +103,13 @@ class CalibLoader
-  /// \tparam T object type to be handled
-  ///
-  /// In case the object is owned the object will be deleted in case the helper goes
-  /// out-of-scope. Otherwise ownership will be assumed with someone else and the object
-  /// will not be deleted in the destructor.
-  template <typename T>
-  class ManagedObject
-  {
-   public:
-    /// \brief Dummy constructor
-    ManagedObject() = default;
-
-    /// \brief Constructor, defining the object as managed or not
-    /// \param object Object to be handled
-    /// \param managed If true the object will be automatically deleted if the helper goes out-of-scope
-    ManagedObject(T* object, bool managed) : mObject(object), mManaged(managed) {}
-
-    /// \brief Destructor, deletes only managed objects
-    ~ManagedObject()
-    {
-      if (mManaged) {
-        delete mObject;
-      }
-    }
-
-    /// \brief Access to the object
-    /// \return Object handled by the helper
-    T* getObject() const { return mObject; }
-
-   private:
-    T* mObject = nullptr;  ///< Object to be handled
-    bool mManaged = false; ///< Management status
-  };
-
-  bool mEnableBadChannelMap;                                         ///< Switch for enabling / disabling loading of the bad channel map
-  bool mEnableTimeCalib;                                             ///< Switch for enabling / disabling loading of the time calibration params
-  bool mEnableGainCalib;                                             ///< Switch for enabling / disabling loading of the gain calibration params
-  std::string mPathBadChannelMap;                                    ///< Path of local file with bad channel map (optional)
-  std::string mPathTimeCalib;                                        ///< Path of local file with time calibration params (optional)
-  std::string mPathGainCalib;                                        ///< Path of local file with gain calibration params (optional)
-  ManagedObject<o2::emcal::BadChannelMap> mBadChannelMap;            ///< Container of current bad channel map
-  ManagedObject<o2::emcal::TimeCalibrationParams> mTimeCalibParams;  ///< Container of current time calibration object
-  ManagedObject<o2::emcal::GainCalibrationFactors> mGainCalibParams; ///< Container of current gain calibration object
+  bool mEnableBadChannelMap;                           ///< Switch for enabling / disabling loading of the bad channel map
+  bool mEnableTimeCalib;                               ///< Switch for enabling / disabling loading of the time calibration params
+  bool mEnableGainCalib;                               ///< Switch for enabling / disabling loading of the gain calibration params
+  o2::emcal::BadChannelMap* mBadChannelMap;            ///< Container of current bad channel map
+  o2::emcal::TimeCalibrationParams* mTimeCalibParams;  ///< Container of current time calibration object
+  o2::emcal::GainCalibrationFactors* mGainCalibParams; ///< Container of current gain calibration object
```

Comment:
You better init them with nullptr, they will be undefined unless finalizeCCDB assigns them.

Reply:
I see, sure, will be fixed immediately.